content/docs/5.1/develop/rawkv/cas.md

```diff
@@ -19,15 +19,15 @@ if (prevValue == request.prevValue) {
 return prevValue;
 ```
 
-The atomicity guarantees that the new value is calculated based on up-to-date information; if the value had been updated by another thread in the meantime, the write would fail.
+The atomicity guarantees that the new value is calculated based on up-to-date information. If the value had been updated by another thread in the meantime, the write would fail.
 
 {{< warning >}}
-CAS can normally prevent problems from concurrent access, but suffers from [ABA problem](https://en.wikipedia.org/wiki/ABA_problem). 
+CAS can normally prevent problems from concurrent access, but suffers from [ABA problem](https://en.wikipedia.org/wiki/ABA_problem).
 {{</ warning >}}
 
 ## Java
 
-The following example shows a simple example of how to use the `CAS` API in java.
+The following example shows how to use `CAS` API in java.
```

Comment:
@Liuxiaozhen12 java --> Java    The same for similar places (not in code) in this PR.